prov/efa/src/efa_prov_info.c

```diff
@@ -77,7 +77,7 @@ const struct fi_domain_attr efa_domain_attr = {
 	.control_progress	= FI_PROGRESS_AUTO,
 	.data_progress		= FI_PROGRESS_AUTO,
 	.resource_mgmt		= FI_RM_DISABLED,
-	.mr_mode		= OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC,
```

Comment:
Is the old code here also a bug that is revealed by the domain squash & info code refactor?

Reply:
Yes. I update the PR and commit message to make the change more clear. The change is to remove `FI_MR_BASIC`. 

This flag is defined only for backward compatibility (for libfabric <= 1.5), and cannot be combined with other mr_mode.

See the section about FI_MR_BASIC in

https://ofiwg.github.io/libfabric/main/man/fi_domain.3.html

It did not cause problem before because `get_lower_rdm_info` called `ofi_alter_info` which removed it, but it should not have been in there.